components/Card.tsx

```diff
@@ -0,0 +1,86 @@
+import styles from "../styles/Card.module.css";
+import Image from "next/image";
+import { User } from "../lib/users";
+
+interface IUserProps {
+	user: User;
+}
+
+export const Card = (user: IUserProps) => {
+	const { fullName, email, avatar, address, gender, age, images } = user.user;
+
+	function printImageButtons() {
```

Comment:
Jak wyżej korzystasz z arrow function to dobrze by było żeby było spójnie :) I nazwałbym to raczej `getImageButtons`